sigstore/verify/models.py

```diff
@@ -438,3 +451,62 @@ def rekor_entry(self, client: RekorClient) -> LogEntry:
             raise InvalidRekorEntry
 
         return entry
+
+    def to_bundle(self) -> Bundle:
+        """Converts VerificationMaterials into a Bundle. Requires that
+        the VerificationMaterials have a Rekor entry loaded. This is
+        the reverse operation of VerificationMaterials.from_bundle()
+        """
+        if not self.has_rekor_entry:
```

Comment:
This makes sense to me, but just to confirm we're on the same page: are you thinking that the CLI that uses this will pre-populate the Rekor entry? I think that's the right way to do this; just making sure 🙂

Reply:
That's how I was thinking about it yes!